### ISSUE TITLE
if 10 folder cross-validation applied on modelnet10 is a good idea?

### ISSUE THREAD

**Issue by shangguan9191** [rusty1s/pytorch_geometric#1570]
how many epoch will be appopriate, since one epoch takes 164s already, for 10 folder will be 1640s per epoch. if 100 epoch is neccesarry?

**Comment by rusty1s** [comment]
Applying 10-fold cross validation on ModelNet10 is rather unusual (at least I haven't seen it used in any papers evaluating on ModelNet10), but it should certainly help for finding better hyperparameters for your model. However, as you mentioned, training times will be pretty long, so it might not be worth it.

**Comment by shangguan9191** [comment]
@rusty1s and quick question regarding [StratifiedKFold](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.StratifiedKFold.html#sklearn.model_selection.StratifiedKFold) . 
for same dataset, If everytime it return a,b,c,d,e folders, instead of a1,b1,c1,d1,e1.  In otherwords, the seed is same.

**Comment by rusty1s** [comment]
Yes, that sounds legit. For `StratifiedKFold`, the splits should only match in case you set a seed/random state.
